ts/parts/Stacking.ts

```diff
@@ -671,6 +671,7 @@ Axis.prototype.cleanStacks = function (this: Highcharts.Axis): void {
 Series.prototype.setGroupedPoints = function (this: Highcharts.Series): void {
     if (
         this.options.centerInCategory &&
+        this.type.match(/column|columnrange|bar/g) &&
```

Comment:
Use `this.is` instead of string type matching, otherwise we won't be able to catch inherited series. Since `bar` inherits `column`, it should be enough to do `(this.is('column') || this.is('columnrange'))`